cleanlab/filter.py

```diff
@@ -66,6 +66,7 @@ def find_label_issues(
     confident_joint=None,
     n_jobs=None,
     verbose=False,
+    allow_missing_classes=False,
```

Comment:
I don't think we should add this argument to our public API (which we can never change in the future once we add an argument). Let's just not have multilabel support missing classes right now. We be supporting missing classes shortly in the package FYI, so you don't have to worry about this. Just make sure your multilabel test does not have missing classes in the meantime.

Reply:
That said if you really need this allow missing functionality for multilabel to work at all, then you can just declare it internally inside your _find_label_issues_multilabel() helper function. The user should not have to know about this though

Reply:
yes I'll fix the test case so it doesn't run into this error